docker/sars_cov_2/primers.sources

```diff
@@ -0,0 +1,3 @@
+source_name,repository,commit,source_schemes
+epi2me-labs,https://github.com/epi2me-labs/wf-artic,a60a1e1e73bde1971c680bd3d53076127dd63fc6,data/primer_schemes/SARS-CoV-2
+quick-lab,https://github.com/quick-lab/SARS-CoV-2,a716b8b5f5a1f31946ddf075e3ec60b09e09096d,.
```

Comment:
Is the full stop intentional at the end here? `.`

Reply:
yes, it is the current directory.